src/main/scala/com/databricks/spark/xml/util/ParseMode.scala

```diff
@@ -1,59 +0,0 @@
-/*
```

Comment:
Duplicate of https://github.com/apache/spark/blob/master/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/ParseMode.scala